compendium/DeclarativeServices/src/manager/states/CCSatisfiedState.cpp

```diff
@@ -24,7 +24,7 @@
 #include "CCUnsatisfiedReferenceState.hpp"
 #include "../ComponentConfigurationImpl.hpp"
 
-using cppmicroservices::service::component::runtime::dto::UNSATISFIED_REFERENCE;
+using cppmicroservices::service::component::runtime::dto::ComponentState;
```

Comment:
Don't need this line since the one place that uses ComponentState is fully qualified?